config/operator/all-in-one/roles.yaml

```diff
@@ -0,0 +1,14 @@
+# permissions needed for Beat cluster-wide autodiscover feature
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: elastic-operator-beat-autodiscover
+rules:
+- apiGroups: [""] # "" indicates the core API group
```

Comment:
This line is deleted by https://github.com/elastic/cloud-on-k8s/blob/57269eca4c64c24599998d0406e720bd305b7a5b/config/operator/Makefile#L25-L26

You should either remove the comment or update the `Makefile`